universal/include/userver/formats/parse/try_parse.hpp

```diff
@@ -14,7 +14,7 @@ namespace formats::parse {
 namespace impl {
 
 template <typename T, typename Value>
-inline bool Is(Value&& value) {
+constexpr inline bool Is(Value&& value) {
   if constexpr(std::is_convertible_v<T, std::int64_t>) {
```

Comment:
This check is imprecise: `IsInt` checks if integer is stored, and it fits in `int`. Instead, for most integer types, `IsInt64` should be used, and for `uint64`, `IsUInt64` should be used. Check the exact signatures against `json::Value`